src/controllers/userController.js

```diff
@@ -41,101 +39,98 @@ const loginUser = async (req, res) => {
         id: user.id,
         email,
         role: user.role,
-        status: user.status,
+        status: user.status
       };
 
       if (user.status == false) {
         return res.status(403).json({ message: 'Account locked!' });
-      } 
-      
+      }
 
-      if (user.role === "seller") {
+      if (user.role === 'seller') {
         const otp = Math.floor(100000 + Math.random() * 900000);
 
         const OTPcontents = {
           userId: user.id,
-          otpCode: otp,
+          otpCode: otp
         };
 
-        const OTPtoken = await generateToken(OTPcontents, process.env.OTP_EXPIRY); //expires in 5 minutes
+        const OTPtoken = await generateToken(OTPcontents, process.env.OTP_EXPIRY); // expires in 5 minutes
 
-        const html=`<h1> Hello</h1>
+        const html = `<h1> Hello</h1>
         <p> <b>${otp}</b> is your OTP code, it expires in 5 minutes so click the button below to verify it. Do not share it with anyone!</p>
         <a href="${process.env.clientURL}/api/v1/users/otp/verify?token=${OTPtoken}" style="background-color:#008CBA;color:#fff;padding:14px 25px;text-align:center;text-decoration:none;display:inline-block;border-radius:4px;font-size:16px;margin-top:20px;">Verify OTP code</a>
-        <p>If you did not register for an account with Falcons Project, please ignore this email.</p>`
+        <p>If you did not register for an account with Falcons Project, please ignore this email.</p>`;
```

Comment:
why I am seeing this HTML content here you should have created it In messages files. then import it here

Reply:
That's just how the helper function was built, there is no other place I could have placed the HTML content